src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -290,6 +290,10 @@ protected Suppression(
             }
         }
 
+        /**
+         * Indicates whether some other object is "equal to" this one.
+         * @noinspection EqualsCalledOnEnumConstant
```

Comment:
reason of suppression to javdoc descrition

Reply:
Done.